framework/src/outputs/FileOutput.C

```diff
@@ -74,12 +77,24 @@ FileOutput::FileOutput(const InputParameters & parameters) :
     _file_base += buffer;
   }
 
-  // Check the file directory of file_base
+  // Check the file directory of file_base and create if needed
   std::string base = "./" + _file_base;
   base = base.substr(0, base.find_last_of('/'));
   if (access(base.c_str(), W_OK) == -1)
-    mooseError("Can not write to directory: " + base + " for file base: " + _file_base);
-
+  {
+    //Directory does not exist. Loop through incremental directories and create as needed.
+    size_t pos = 0;
+    std::string inc_path;
+    do
```

Comment:
I suspect that we have many places throughout the framework that don't deal with `\/` properly (file / directory handling for restart raises its hand).

This is actually another reason _TO_ use `tokenize()`... because if we make it deal with escaped characters properly in the future then it will fix this too 😄